server/sonar-server/src/main/java/org/sonar/server/activity/index/ActivityIndex.java

```diff
@@ -67,35 +65,33 @@ public SearchResponse doSearch(ActivityQuery query, SearchOptions options) {
     requestBuilder.setSize(options.getLimit());
     requestBuilder.addSort(ActivityIndexDefinition.FIELD_CREATED_AT, SortOrder.DESC);
 
-    AndFilterBuilder filter = FilterBuilders.andFilter();
+    AndQueryBuilder filter = QueryBuilders.andQuery();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "AndQueryBuilder"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "andQuery"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)